File: JView/sample2/sample2.js
jdata {
    float temp as logger;
    float pos as logger;
}

setInterval(function() {

	for (i = 0; i < pos.size(); i++) {
	    if (pos[i] !== undefined && pos[i].lastValue() != null)
	        console.log("Pos: ", pos[i].lastValue());
	}

	for (i = 0; i < temp.size(); i++) {
	    if (temp[i] !== undefined && temp[i].lastValue() != null)
		console.log("i ", i, " Temp: ", temp[i].lastValue());
	}
    }, 5000);

File: JView/sample2/sample2.c
#include <stdlib.h>

int main() {

    while (1) {
        float q = rand() % 100 + 0.1 * (rand() % 10);
        temp = q;
	float p = rand() % 100 * 0.1;
	pos = p;
	//	printf("temp = %f, pos = %d\n", q, p);
        sleep(5);
    }
}
